app/services/ingest_csv.rb

```diff
@@ -38,6 +38,8 @@ def call
         else
           # Skip row and write message
           write_log("Skipping Activity #{row}: #{activity.errors.full_messages}")
+          write_log("attributes: #{attributes.inspect}")
+          # binding.pry
```

Comment:
Don't know if you are intending to keep this?

Reply:
Yep, it's there for a reason 👍